resolwe_bio/tools/multiqc_sample_info.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+"""Format sample metadata table for MultiQC."""
+import argparse
+import json
+
+
+sample_summary_json = {
+    "id": "sample_info",
+    "section_name": "Sample Info",
+    "plot_type": "table",
+    "file_format": "json",
+    "data": {}
+}
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="Parse library type information.")
+    parser.add_argument('--samples', nargs='+', help="Samples.")
+    parser.add_argument('--species', nargs='+', help="Species.")
+    parser.add_argument('--build', nargs='+', help="Build.")
+    return parser.parse_args()
+
+
+def main():
+    """Invoke when run directly as a program."""
+    args = parse_arguments()
+
+    for sample in zip(args.samples, args.species, args.build):
+        sample_name, species, build = sample
+        if build not in ['rRNA', 'globin']:
+             sample_summary_json['data'][sample_name] = {
+                 'Species': species,
```

Comment:
Is this meant to be "human readable" (space in genome build, capital letters)?

Reply:
Yes, those are labels in the final rendered table.